### PR TITLE
feat(addon): enable materia kv

### PR DESCRIPTION
**Pull request by aurrelhebert**

### CODE REVIEW

**Comment by hsablonniere** [comment]
It would be nice to find a way not to have a default plan.

**Comment by hsablonniere** [comment]
image

* I would add an empty line after `Name:` and before the warning
* I would spell it `MateriaDB KV` instead of `materia kv` in the warning
* I would use yellow for the warning and another color (green or blue) for the commands
* Maybe add an ascii warning before the message like this `/!\`

Apart from those details, well done @aurrelhebert

File: src/commands/addon.js [hunk 1]
```diff
@@ -98,6 +101,13 @@ function displayAddon (format, addon, message) {
         `Real ID: ${addon.realId}`,
         `Name: ${addon.name}`,
       ].join('\n'));
+      if (providerName === 'kv') {
+        const materia = `The materia ${providerName} provider is currently available in alpha, consider using it with care \n`
+          + 'To use the redis-cli command exec:\n'
+          + colors.yellow(`source <(clever addon env ${addon.id} -F shell)\n`)
+          + colors.yellow('redis-cli -h $KV_HOST -p $KV_PORT');
+        Logger.println(materia);
+      }
```
Comment (hsablonniere) [inline src/commands/addon.js:110]: suggestion: sometimes I use an array + join to simplify multiline messages

```js
if (providerName === 'kv') {
  const materiaMessage = [
    `The materia ${providerName} provider is currently available in alpha, consider using it with care`,
    'To use the redis-cli command exec:',
    colors.yellow(`source <(clever addon env ${addon.id} -F shell)`),
    colors.yellow('redis-cli -h $KV_HOST -p $KV_PORT'),
  ].join('\n');
  Logger.println(materia);
}
```

**Review by hsablonniere — CHANGES_REQUESTED** [review]
I added a suggestion and other comments are above.

**Review by hsablonniere — APPROVED** [review]
LGTM :+1: 

TODO: fixup both commits into one

File: src/commands/addon.js [hunk 2]
```diff
@@ -70,23 +87,26 @@ async function create (params) {
...
     case 'json': {
-      Logger.printJson({
+      const jsonAddon = {
         id: addon.id,
         realId: addon.realId,
         name: addon.name,
-      });
+      };
+      Logger.printJson((providerName === 'kv')
+        ? { ...jsonAddon, availability: 'alpha', warning: 'Consider using it with care' }
```
Comment (davlgd) [inline src/commands/addon.js:108]:
```suggestion
        ? { ...jsonAddon, availability: 'alpha', warning: 'Don't store sensitive or production grade data' }
```

File: src/commands/addon.js [hunk 3]
```diff
@@ -98,6 +118,16 @@ function displayAddon (format, addon, message) {
         `Real ID: ${addon.realId}`,
         `Name: ${addon.name}`,
       ].join('\n'));
+      if (providerName === 'kv') {
+        const materiaMessage = [
+          '',
+          colors.yellow(`/!\\ The MateriaDB ${providerName.toUpperCase()} provider is currently available in alpha, consider using it with care`),
```
Comment (davlgd) [inline src/commands/addon.js:124]:
```suggestion
          colors.yellow(`/!\\ MateriaDB ${providerName.toUpperCase()} is in Alpha testing phase, don't store sensitive or production grade data`),
```

Comment (davlgd) [reply src/commands/addon.js:124]: `/!\` or ⚠️ ? :D

File: src/commands/addon.js [hunk 4]
```diff
@@ -98,6 +118,16 @@ function displayAddon (format, addon, message) {
...
+          'To use the redis-cli command exec:',
```
Comment (davlgd) [inline src/commands/addon.js:125]:
```suggestion
          "You can easily use MateriaDB KV with 'redis-cli', with such commands:",
```

**Review by davlgd — CHANGES_REQUESTED** [review]
LGTM, just some text nitpicks/sync with Console

**Review by davlgd — APPROVED** [review]
LGTM, Good team work! 💡☁️🚀

**Pull request merged**